src/query/functions/src/scalars/geometry.rs

```diff
@@ -577,6 +551,110 @@ pub fn register(registry: &mut FunctionRegistry) {
         }),
     );
 
+    registry.register_passthrough_nullable_1_arg::<StringType, GeometryType, _, _>(
+        "to_geometry",
```

Comment:
We can also support the `try_to_geometry` function by the way, which returns NULL instead of reporting an error when it is encountered.